teamwork/apps/profiles/models.py

```diff
@@ -50,6 +50,38 @@ def save(self, *args, **kwargs):
         """
         super(Skills, self).save(*args, **kwargs)
 
+#-----------------------------------------------------------------------
+class Techs(models.Model):
```

Comment:
Can you move this Techs model from `apps/profiles/models.py` to `apps/projects/models.py`